src/person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(name = 'Unknown', age = 0, parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- [  OPTIONAL ] Hey there! 🚀 I was thinking, wouldn't it be awesome if we moved those getters and setters right up to the top of the class? 📦🚀 What do you reckon? 🤔 Let's aim to declare the getters and setters at the top of the class for maximum clarity and organization! 🌟👩‍💻🌈